### PR TITLE
docs: updating screenshots and reworking scenarios descriptions for tasklist 8.2

### PR DESCRIPTION
**Pull request by marcosgvieira**
## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._

As we added more fields and changed some screens in tasklist, updating screenshots and adding descriptions on what have changed

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 1]
```diff
@@ -74,3 +78,31 @@ To add a new variable, click **Add Variable**.
 You will now see the completed task by selecting the **Completed** task list:
 
 ![tasklist-task-completed](img/tasklist-task-completed_light.png)
+
+### Processes
+
+> **_NOTE:_** This tab is only available for Self-Managed environments at the moment
```
Comment (vsgoulart) [inline docs/components/tasklist/userguide/using-tasklist.md:84]:
```suggestion
> **_NOTE:_** This tab is only available for Self-Managed environments or an alpha cluster at Camunda Platform  at the moment
```

**Review by vsgoulart — CHANGES_REQUESTED** [review]

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit 👍

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 2]
```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
...
 
 - All open
-- Claimed by me
-- Unclaimed
+- Assigned to me
+- Unassigned
 - Completed
 
-Initially, we have no **Claimed by me** tasks.
+You can also order the tasks on the left menu, clicking on the icon ![order-icon](img/order-icon.png):
```
Comment (christian-konrad) [inline docs/components/tasklist/userguide/using-tasklist.md:36]: Rephrase to

"Click on the icon order-icon to order the tasks. You can order them by the date of creation, the due date, or the follow up date.

The follow up date defines the latest time you should start working on a task, helping you to prioritize work.
The due date provides a deadline when the task should be finished."

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 3]
```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
 You can choose which tasks you want to see:
```
Comment (christian-konrad) [inline docs/components/tasklist/userguide/using-tasklist.md:29]: Rephrase to 

"Click on the selection field in the left panel to choose which tasks you want to see."

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 4]
```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
...
-Initially, we have no **Claimed by me** tasks.
+You can also order the tasks on the left menu, clicking on the icon ![order-icon](img/order-icon.png):
 
-![tasklist-claimed-by-me-empty](img/tasklist-claimed-by-me-empty_light.png)
+![tasklist-task-ordering](img/tasklist-task-ordering.png)
+
+### Assign tasks
 
-### Claimed by me tasks
+Initially, we have no **Assigned to me** tasks.
```
Comment (christian-konrad) [inline docs/components/tasklist/userguide/using-tasklist.md:42]: Rephrase to "When no tasks are assigned to you, the list appears empty."

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 5]
```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
...
 
-Select the **Unclaimed** list and claim a task using the **Claim** button on the details panel:
+![tasklist-claimed-by-me-empty](img/tasklist-claimed-by-me-empty_light.png)
+
+Select the **Unassigned** list and assign a task using the **Assign to me** button on the top panel:
```
Comment (christian-konrad) [inline docs/components/tasklist/userguide/using-tasklist.md:46]: Rephrase to "Select the **Unassigned** list and assign a task to yourself using the **Assign to me** button on the top panel."

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 6]
```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
...
 
 ![tasklist-claim](img/tasklist-claim_light.png)
 
-### Claim a task
+### Assigned to me tasks
```
Comment (christian-konrad) [inline docs/components/tasklist/userguide/using-tasklist.md:50]: Rephrase to "Work on assigned tasks"

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 7]
```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
...
 
-Select the **Claimed by me** list to see if you claimed the task:
+Select the **Assigned to me** list to see if you assigned the task:
```
Comment (christian-konrad) [inline docs/components/tasklist/userguide/using-tasklist.md:52]: Rephrase to "Select the **Assigned to me** list to see the tasks that are assigned to you. Select a task to work on it."

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 8]
```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
...
 
 ![tasklist-claimed-by-me-list](img/tasklist-claimed-by-me-list_light.png)
 
 ### Complete a task
 
-Once you've claimed a task, you can complete the task by adding and updating variables, and using the **Complete Task** button:
+Once you are assigned a task, you can complete the task by adding and updating variables, and using the **Complete Task** button:
```
Comment (christian-konrad) [inline docs/components/tasklist/userguide/using-tasklist.md:58]: Rephrase to "When a task is assigned to you, you can complete the task by filling out the given form, and clicking on the **Complete Task** button. There are also cases where no form is available. In these cases, you have to add and/or update variables directly."

**Review by christian-konrad — CHANGES_REQUESTED** [review]
I requested quite some changes in the text. I left comments for proposals to update the paragraphs.

Please also update the title of the PR since it also covers text changes, not just screenshot changes.

**Comment by marcosgvieira** [comment]
@christian-konrad can you please review again?

**Review by vsgoulart — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Pull request merged**